src/parser/core/Module.ts

```diff
@@ -12,7 +12,7 @@ export interface Options {
 class Module {
   static dependencies: { [desiredPropName: string]: typeof Module } = {};
 
-  protected readonly owner!: CombatLogParser;
+  public readonly owner!: CombatLogParser;
```

Comment:
What required this change?

Reply:
There are non-analyzer modules that have the analyzer passed down to them. The Atonement Healing Breakdown uses this variable, but it pulls it from the parent that creates the module. There are a number of modules that do something similar, and I didn't see too much of a stink from making it public instead of protected. I can modify that module to make it fetch the owner some other way if you think I should.